janusgraph-examples/example-remotegraph/README.md

```diff
@@ -8,7 +8,8 @@ multiple clients to connect to the same JanusGraph instance. It also allows
 clients to connect via non-JVM languages. By default, the Gremlin Server
 communicates over WebSocket.
 
-A Java program can connect to the Gremlin Server in two ways:
+[`RemoteGraphApp.java`](src/main/java/org/janusgraph/example/RemoteGraphApp.java)
+is an example Java program that connects to the Gremlin Server in two ways:
 * [Gremlin Driver](http://tinkerpop.apache.org/docs/3.2.6/reference/#connecting-via-java)
```

Comment:
I think the TinkerPop links aren't automatically updated here as in the ascii docs? Can you create another issue to look at options for helping maintain currency of TinkerPop links in these and other relevant (Markdown) docs?

Reply:
opened #582